src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -291,6 +290,10 @@ else if (sipCall != null)
                 }
             }
         }
+        if(gatewaySession instanceof TranscriptionGatewaySession)
+        {
+            mucDisplayName = TranscriptionGatewaySession.DISPLAY_NAME;
+        }
```

Comment:
Thanks for renaming this! I would refactor a bit more and move the method to GatewaySession (it already has the callContext) to avoid casting.